### DISCUSSION TITLE
Request: Parameter to change inject_args replacement target strings (currently $1 for Unix)

### DISCUSSION THREAD

**Discussion by AndydeCleyre** [espanso/espanso#1185] [Feature requests and Ideas]
Especially when using a `shell` match, there's a real chance of clashing with external-to-espanso uses of `$1`, `$2`, etc. For examples, see `rg -r` (ripgrep) and `pcre2grep -O`.

Some alternative strings which might be handy:
- `##1##`
- `{{1}}`

Bash-y and regex-y syntaxes should be avoided if this leads to a hardcoded set of alternatives.

---

EDIT: A couple of example `cmd`s for `shell` matches which won't work without this, intending to expand a local ipv4 address:

```yaml
cmd: 'ip addr | pcre2grep -M -O ''$2'' ''^\d: wlan0: (.*\n[^\d])*.* inet ((\d+\.){3}\d+)/'''
cmd: 'nmcli device show wlan0 | pcre2grep -M -O ''$1'' ''^IP4.ADDRESS\[1\]: *(.*)/.*'''
```

**Comment by federico-terzi** [comment]
Thanks @AndydeCleyre, I'll add it to the roadmap :)
